cmd/agent/app/integrations.go

```diff
@@ -135,6 +136,23 @@ func getTUFConfigFilePath() (string, error) {
 	return tufConfig, nil
 }
 
+func validateTufArgs(args []string) error {
+	if len(args) > 1 {
+		return fmt.Errorf("Too many arguments")
+	}
+
+	exp, err := regexp.Compile(tufPkgPattern)
+	if err != nil {
+		return fmt.Errorf("internal error: %v", err)
+	}
+
+	if !exp.MatchString(args[0]) {
```

Comment:
we should also check beforehand that `len(args) > 0`, otherwise when no integration name is provided this index is out of range and a panic is raised

Reply:
That's the first thing I check at the top of the function :)